packages/create-sitecore-jss/src/templates/nextjs-styleguide-tracking/data/dictionary/en.yml

```diff
@@ -0,0 +1,5 @@
+styleguide-sample: This is a dictionary entry in English as a demonstration
```

Comment:
Should we add [`{{language}}.yml`](https://github.com/Sitecore/jss/blob/dev/packages/create-sitecore-jss/src/templates/nextjs-styleguide/data/dictionary/%7B%7Blanguage%7D%7D.yml) override as well?

Reply:
In this case, we will need to have information about `language` in the `styleguide-tracking` initializer, but we don't have it
and `styleguide-tracking` component is wasn't implemented within another language, we had it only inside the EN page, we will have empty `/tracking` page in this case

Reply:
You are right, this would only be useful if we had translated `styleguide-tracking` component/page. Forget it then :)